tests/format/scalafmt/test

```diff
@@ -23,36 +23,36 @@ bazel run :unformatted-binary-copy.format
 bazel run :unformatted-binary-copy.format-test
 rm unformatted.tmp.scala
 
-# non-default output format test: library
-## check
-bazel run :formatted-library.format-test
-! bazel run :unformatted-library.format-test
-## copy
-cp unformatted.template.scala unformatted.tmp.scala
-! bazel run :unformatted-library-copy.format-test
-bazel run :unformatted-library-copy.format
-bazel run :unformatted-library-copy.format-test
-rm unformatted.tmp.scala
+# # non-default output format test: library
+# ## check
+# bazel run :formatted-library.format-test
+# ! bazel run :unformatted-library.format-test
+# ## copy
+# cp unformatted.template.scala unformatted.tmp.scala
+# ! bazel run :unformatted-library-copy.format-test
+# bazel run :unformatted-library-copy.format
+# bazel run :unformatted-library-copy.format-test
+# rm unformatted.tmp.scala
```

Comment:
are all these things commented out on purpose?

Reply:
Nope. Pretty sure that's my bad. I'll go fix this.

Reply:
I've uncommented this locally and the test is passing. I'll push it up with the other requested changes.